libraries/botbuilder-adapter-twilio-whatsapp/src/twilioWhatsAppAdapter.ts

```diff
@@ -57,24 +94,51 @@ export class TwilioWhatsAppAdapter extends BotAdapter {
     protected readonly client: Twilio.Twilio;
     protected readonly channel: string = 'whatsapp';
 
+    // OAuth Properties
+    protected readonly credentials: AppCredentials;
+    protected readonly credentialsProvider: SimpleCredentialProvider;
     /**
      * Creates a new TwilioWhatsAppAdapter instance.
      * @param settings configuration settings for the adapter.
      */
-    public constructor(settings: TwilioWhatsAppAdapterSettings) {
+    public constructor(settings?: Partial<TwilioWhatsAppAdapterSettings>) {
```

Comment:
What about adding a second parameter to the constructor, which is the default `BotFrameworkAdapterSettings`? Or is this too much overhead?

Would be great if we could use the default adapter interface, as an optional constructor parameter.